packages/core/src/browser/pageExitObservable.ts

```diff
@@ -0,0 +1,35 @@
+import { Observable } from '../tools/observable'
+import { addEventListener, DOM_EVENT } from '../tools/utils'
+
+export interface PageExitEvent {
+  isUnloading: boolean
+}
+
+export function createPageExitObservable(): Observable<PageExitEvent> {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Love it !